spec/models/manageiq/providers/vmware/infra_manager/provision_workflow_spec.rb

```diff
@@ -12,42 +14,73 @@
 
   describe "#new" do
     it "pass platform attributes to automate" do
-      MiqProvisionWorkflow.any_instance.stub(:get_dialogs).and_return(:dialogs => {})
-      MiqAeEngine::MiqAeWorkspaceRuntime.should_receive(:instantiate)
-      MiqAeEngine.should_receive(:create_automation_object) do |name, attrs, _options|
-        name.should eq("REQUEST")
-        attrs.should have_attributes(
-          'request'                   => 'UI_PROVISION_INFO',
-          'message'                   => 'get_pre_dialog_name',
-          'dialog_input_request_type' => 'template',
-          'dialog_input_target_type'  => 'vm',
-          'platform_category'         => 'infra',
-          'platform'                  => 'vmware'
-        )
-      end
+      stub_dialog(:get_dialogs)
+      assert_automate_dialog_lookup("infra", "vmware", "get_pre_dialog_name", nil)
 
       described_class.new({}, admin.userid)
     end
+
+    it "pass platform attributes to dialog_name" do
+      stub_dialog(:get_pre_dialogs)
+
+      described_class.new({}, admin.userid)
```

Comment:
Is this test missing an assertion or will it raise an error if platform is not passed?